lib/models/playlist.dart

```diff
@@ -100,13 +100,14 @@ class Playlist {
   String? description;
   int videoCount;
   int? viewCount;
-  bool? isListed ;
+  bool? isListed;
+  int updated;
```

Comment:
I think updated should be optional.

When I make a search it crashes because updated doesn't exist:
![image](https://github.com/lamarios/clipious/assets/1192563/4c970501-f4b8-437d-b3aa-e583917bfaea)

Reply:
Apologies. Fixed this